.github/scripts/check-yaml.py

```diff
@@ -253,12 +247,65 @@ def check(self) -> int:
                             message="The file must be non-empty",
                         )
 
+                if "document" in parsed:
+                    repo = parsed["document"].get("repo")
+                    commit = parsed["document"].get("commit")
```

Comment:
We also need to validate the commit value is a valid SHA value. This is probably best in the schema were we can use a regex pattern.

https://json-schema.org/understanding-json-schema/reference/string#regexp

Reply:
I opened https://github.com/instructlab/schema/issues/30 for this.

Reply:
right, that is probably the most/more cost effective change here